### PR TITLE
Add permissions for insightappsec-plugin

### PR DESCRIPTION
**Pull request by r-powell-r7**
# Description

https://github.com/jenkinsci/insightappsec-plugin
https://issues.jenkins-ci.org/browse/HOSTING-695

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

File: permissions/plugin-insightappsec.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "insightappsec"
+github: "jenkinsci/insightappsec-plugin"
+paths:
+- "org/jenkins-ci/plugins/insightappsec"
+developers:
+- "ias_developer"
```
Comment (batmat) [inline permissions/plugin-insightappsec.yml]: so you're not allowing `supersloth` to release, I guess on purpose right?

File: permissions/plugin-insightappsec.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "insightappsec"
+github: "jenkinsci/insightappsec-plugin"
+paths:
+- "org/jenkins-ci/plugins/insightappsec"
```
Comment (batmat) [inline permissions/plugin-insightappsec.yml]: Side note: we tend to recommend people to now prefer `io.jenkins.plugins`, but not a requirement.

File: permissions/plugin-insightappsec.yml (hunk 1 above)
Comment (batmat) [reply permissions/plugin-insightappsec.yml]: So given https://issues.jenkins-ci.org/browse/HOSTING-695?focusedCommentId=358110&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-358110 it does look intended. :+1:

**Comment by r-powell-r7** [comment]
Hey @batmat , I updated the `paths`. Also updated the `groupId` in plugins pom https://github.com/jenkinsci/insightappsec-plugin/blob/master/pom.xml

If this looks good can you go ahead and merge this please? 
Thanks

**Comment by batmat** [comment]
Looks good, moving forward.

**Pull request merged**